<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/bg.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-10</y>
      <width>631</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/b32adf3c70abfdf29c912c0cffd7e061.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="cntime">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Time">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Score">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>7</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="drumR1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/3333.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="drumR2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/3333.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="drumB1">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>100</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/111.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="drumR3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/3333.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="drumB2">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>100</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/111.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="drumB3">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>100</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/111.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="scorevalue">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="drum">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>91</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/7777.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="tiRed">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/3333.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="tiBlu">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/111.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="J">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>J</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="K">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>K</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="finalsco">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>85</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="finaltext">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FinalScore:</string>
    </property>
   </widget>
   <widget class="QLabel" name="drumB4">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>100</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/111.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="drumB5">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>100</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/111.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="drumR4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/3333.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="drumR5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="surface.qrc">:/Surface/3333.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>background</zorder>
   <zorder>Score</zorder>
   <zorder>Time</zorder>
   <zorder>cntime</zorder>
   <zorder>scorevalue</zorder>
   <zorder>drum</zorder>
   <zorder>drumB1</zorder>
   <zorder>drumB2</zorder>
   <zorder>drumB3</zorder>
   <zorder>drumR1</zorder>
   <zorder>drumR2</zorder>
   <zorder>drumR3</zorder>
   <zorder>tiBlu</zorder>
   <zorder>tiRed</zorder>
   <zorder>J</zorder>
   <zorder>K</zorder>
   <zorder>finalsco</zorder>
   <zorder>finaltext</zorder>
   <zorder>drumB4</zorder>
   <zorder>drumB5</zorder>
   <zorder>drumR4</zorder>
   <zorder>drumR5</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="surface.qrc"/>
 </resources>
 <connections/>
</ui>
